pipeline/data/cjk.py

```diff
@@ -102,3 +134,50 @@ def _convert_line(self, text: str, to: ChineseType) -> str:
         elif to == ChineseType.traditional:
             return self.s2t.convert(text)
         raise ValueError(f"Unsupported type: {to}")
+
+
+def handle_chinese_mono(file_destination: Path, is_src: bool, variant: ChineseType):
+    converted_path = file_destination.with_suffix(".converted.zst")
+    chinese_converter = ChineseConverter()
+    if is_src:
+        logger.info(f"Converting the output file to {variant}")
+        stats = chinese_converter.convert_file(file_destination, converted_path, variant)
+    else:
+        logger.info(f"Filtering out everything except {variant} in the output file")
+        stats = chinese_converter.filter_file(file_destination, converted_path, variant)
+    shutil.move(converted_path, file_destination)
```

Comment:
Stylistically, since we're using `pathlib`, I'd prefer to use its methods like `.rename` and `.unlink` rather than using multiple libraries like `shutil`, unless there is a good reason to do so. I generally prefer `pathlib` over other file and file path manipulation tools, as it's pretty ergonomic to work with.

Reply:
`.replace` similar to `.move` does it in one line